controlplane/cmd/monitor-client.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
Could we put these in examples/skydive/ rather than in controlplane?

Reply:
Yep, no problem

Reply:
@matrohon Just represented you to reply to Ed (I think you won't mind that haha)